roles/lib/files/FWO.Services/ModellingVarianceAnalysis.cs

```diff
@@ -445,9 +670,82 @@ private void FillGroupMembers(ModellingAppRole appRole, Management mgt)
             }
         }
 
+        private void FillGroupMembersAppZone(ModellingAppZone appZone, Management mgt)
```

Comment:
use FillGroupMembers (after changing parameter appRole into idstring, as only this is used inside) also here instead of duplicating the whole code